rocAL/rocAL_pybind/examples/image_comparison.py

```diff
@@ -0,0 +1,148 @@
+from PIL import Image
+import os
+import sys
+import datetime
+import logging
+
+def compare_pixels(img1, img2, aug_name, width, height, image_offset = 0):
+    pixel_difference = [0,0,0,0,0,0]
+    if'rgb' in aug_name:
+        pixels1 = img1.load()
+        pixels2 = img2.load()
+        total_valid_pixel_count = width * height * 3
+        for wt in range(width):
+            for ht in range(height):
+                ht = ht + image_offset
+                if pixels1[wt,ht] != pixels2[wt,ht]:
+                    r = abs(pixels1[wt,ht][0]-pixels2[wt,ht][0])
```

Comment:
Please add space before and aftter - operator along with space after comma
Applicable to all such instances in the file